zerver/tests/test_events.py

```diff
@@ -1326,6 +1329,20 @@ def test_realm_domain_events(self) -> None:
         check_realm_domains_remove("events[0]", events[0])
         self.assertEqual(events[0]["domain"], "zulip.org")
 
+    def test_realm_playground_events(self) -> None:
+        playground_info = dict(
+            name="Python playground",
+            pygments_language="Python",
+            url_prefix="https://python.example.com",
+        )
+        events = self.verify_action(
+            lambda: do_add_realm_playground(self.user_profile.realm, playground_info)
+        )
+        check_realm_playgrounds("events[0]", events[0])
+
+        events = self.verify_action(lambda: do_remove_realm_playground(self.user_profile.realm, 1))
```

Comment:
Let's not hardcode 1 here -- that'll break if we adjust `populate_db` to configure some (which we probably will want to do later in this feature's development, to make manual testing more convenient).  Instead, we should save the ID from the previous step.  (Or `last_id = RealmPlayground.objects.last().id`, if that's annoying with the lambda structure)